case/src/main/kotlin/com/ritense/case/web/rest/CaseDefinitionResource.kt

```diff
@@ -148,4 +154,22 @@ class CaseDefinitionResource(
         service.deleteCaseListColumn(caseDefinitionName, columnKey)
         return ResponseEntity.noContent().build()
     }
+
+    @GetMapping("/management/v1/case/{caseDefinitionName}/{caseDefinitionVersion}/export",
+        produces = [MediaType.APPLICATION_OCTET_STREAM_VALUE]
+    )
+    @RunWithoutAuthorization
+    fun getExport(
+        @PathVariable caseDefinitionName: String,
+        @PathVariable caseDefinitionVersion: Long,
+    ): ResponseEntity<ByteArray> {
+        val baos = exportService
+            .createExport(JsonSchemaDocumentDefinitionId.existingId(caseDefinitionName, caseDefinitionVersion))
+        val fileName = caseDefinitionName + "_" + caseDefinitionVersion + "_" +
```

Comment:
Nitpicking: you complete the filename by adding the extension in the header. Might be a bit counter intuitive.